src/validate_pyproject/cli.py

```diff
@@ -106,13 +106,18 @@ def critical_logging():
         dest="tool",
         help="External tools file/url(s) to load, of the form name=URL#path",
     ),
+    "store": dict(
+        flags=("--store",),
+        help="Load a pyproject.toml file and read all the $ref's into tools",
```

Comment:
This file is a `pyproject.json` instead of `toml`, right? The format used by Schema Store, isn't it?

(Sorry if it is a confusion on my part. The colliding file names mess with my brain a bit)

Reply:
Yep, my mistake!

Reply:
```suggestion
        help="Load a pyproject.json file and read all the $ref's into tools",
```